interpreter_book/src/Makefile

```diff
@@ -0,0 +1,17 @@
+all: check test race
```

Comment:
nice idea to add a makefile!

Reply:
thank you for the feedback! :pray: 
it's much easier to type `make monkey` than remember all the intricacies of go build system (at least for me)

Reply:
:heavy_plus_sign: this is awesome!

Reply:
:pray: